wemod_launcher/consts.py

```diff
@@ -74,20 +84,28 @@ def get_compat() -> str:
     return ecompat
 
 
-BASE_STEAM_COMPAT = get_compat()
-STEAM_COMPAT_FOLDER = os.path.dirname(BASE_STEAM_COMPAT)
+def get_scan_folder() -> str:
+    scan_folder: str
+    try:
+        scan_folder = os.getenv("SCANFOLDER") or ""
```

Comment:
im not shure if os.getenv raises errors,
since os.getenv always gives a result,
if there is no entry for then enviroment var it prints none or whatever in the second parrameter,
therefore:
scan_folder = os.getenv("SCANFOLDER", "")
will have "" most of the time
if we want a try block os.environ is a better choice

Reply:
We can ignore this file for now. I'm moving the consts into `utils/consts.py`, so gradually you'll see commits adding functionality there instead.